Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.yml

```diff
@@ -0,0 +1,409 @@
+      description: The country in which the IP address is located.
+      type: string
+    - contextPath: IP.Geo.Description
+      description: Additional information about the location such as city and region.
+      type: string
+    - contextPath: IP.Malicious.Vendor
+      description: The vendor reporting the IP address as malicious.
+      type: string
+    - contextPath: IP.Malicious.Description
+      description: A description explaining why the IP address was reported as malicious.
+      type: string
+  - arguments:
+    - default: true
+      description: List IP addresses to retrieve quick check about
+      isArray: true
+      name: ip
+      required: true
+      secret: false
+    description: 'Check whether a given IP address is "Internet background noise",
+      or has been observed scanning or attacking devices across the Internet. Note:
+      It checks against the last 60 days of Internet scanner data.'
+    execution: false
+    name: greynoise-ip-quick-check
+    outputs:
+    - contextPath: GreyNoise.IP.address
+      description: The IP address of the scanning device IP.
+      type: string
+    - contextPath: GreyNoise.IP.noise
+      description: Whether the IP is internet background noise or attacking.
+      type: boolean
+    - contextPath: GreyNoise.IP.code
+      description: Code which correlates to why GreyNoise labeled the IP as noise.
+      type: string
+    - contextPath: GreyNoise.IP.code_value
+      description: Message which correlates to why GreyNoise labeled the IP as noise.
+      type: string
+  - arguments:
+    - default: false
+      description: "The date the device was most recently observed by GreyNoise. Example: 1d, 2d, 12h, or 1m."
+      name: last_seen
+      required: false
+      secret: false
+    - default: false
+      description: "The organization that owns the network that the IP address belongs to."
+      name: organization
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Classification of the device like unknown, benign, malicious.
+      isArray: false
+      name: classification
+      predefined:
+        - unknown
+        - benign
+        - malicious
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: false
+      description: Whether the IP is spoofable or not.
+      isArray: false
+      name: spoofable
+      predefined:
+        - "true"
+        - "false"
+      required: false
+      secret: false
+    - default: false
+      description: The benign actor the device has been associated with.
+      isArray: false
+      name: actor
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '10'
+      description: Maximum amount of results to grab.
+      isArray: false
+      name: size
+      required: false
+      secret: false
+    - default: false
+      description: "GNQL query to filter records.\n
```

Comment:
Does the common product user know where to find the syntax of such queries?

Reply:
@guykeller, we have added a document in README.md file.
- The document also contains deep link of the GreyNoise API document page which covers more detail on how to write a GNQL query.